Makefile

```diff
@@ -56,7 +52,7 @@ benchmark:
 	$(GOTEST) -bench=. -run=notests $(ALL_PKGS)
 
 .PHONY: travis-ci
-travis-ci: fmt impi vet lint goimports misspell staticcheck test-with-cover otelcol
+travis-ci: impi lint misspell test-with-cover otelcol
```

Comment:
Why are fmt,vet,goimports,staticcheck removed?

Reply:
they are run by umbrella tool.

Reply:
OK. Please modify PR description/commit message to explain that.

Reply:
Sure, updating.